src/applications/proxy-rewrite/utilities/get-asset-path.js

```diff
@@ -0,0 +1,18 @@
+import buckets from 'site/constants/buckets';
```

Comment:
This lives in the platform/site-wide folder so in the interest of Teamsites code isolation, it is duplicated here now.